src/config.ts

```diff
@@ -202,6 +207,11 @@ const getServerSideConfig = (): ConfigType => {
     translateServiceUrl: getEnvironmentVariabel("NDKM_URL", getTranslateServiceUrl(ndlaEnvironment)),
     isVercel: getEnvironmentVariabel("IS_VERCEL", "false") === "true",
     runtimeType: getEnvironmentVariabel("NODE_ENV", "development") as "test" | "development" | "production",
+    aiModelID: getEnvironmentVariabel("NDLA_AI_MODEL_ID", ""),
+    aiRegion: getEnvironmentVariabel("NDLA_AI_MODEL_REGION", ""),
+    aiSecretKey: getEnvironmentVariabel("NDLA_AI_SECRET_KEY", ""),
+    aiSecretID: getEnvironmentVariabel("NDLA_AI_SECRET_ID", ""),
+    transcriptionBucketName: getEnvironmentVariabel("S3_TRANSCRIPTION_BUCKET_NAME", ""),
```

Comment:
Ikke ha de her heller, da blir de eksponert til klient! Hent de ut der det brukes, jeg ville bare ikke at de skulle hentes ut for hver eneste request!